nginx/conf/nginx.conf

```diff
@@ -0,0 +1,50 @@
+
+worker_processes  1;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+    sendfile        on;
+    keepalive_timeout  65;
+
+    server {
+        listen       8000 default_server;
+        server_name  localhost gtkm.com www.gtkm.com;
+        root         ..\gtkm\static;
+
+        location /api/auth/ {
+            proxy_pass http://localhost:8001;
+        }
+
+        location /api/stats/ {
+            proxy_pass http://localhost:8004;
+        }
+    }
+
+    server {
+        listen 8001;
+        listen 8002;
+        listen 8003;
+        listen 8004;
+        
+        location /auth/ {
+            proxy_pass http://localhost:8001;
+        }
+        
+        location /github/ {
+            proxy_pass http://localhost:8002;
+        }
+        
+        location /gitlab/ {
+            proxy_pass http://localhost:8003;
+        }
+        
+        location /stats/ {
+            proxy_pass http://localhost:8004;
+        }
+    }
```

Comment:
To też jest raczej do wywalenia.

Jak już to było opisane w dokumentacji https://github.com/mpfmorawski/git-to-know-me/issues/1#issuecomment-1098813687

> Endpointy zaczynające się na /api/ powinny być dostępne z zewnątrz, pozostałe są tylko na użytek komunikacji między mikroserwisami

A więc pozostałe endpointy nie muszą być obsługiwane przez nginxa